packages/cli-helpers/src/auth/authTasks.ts

```diff
@@ -373,3 +424,59 @@ export const installPackages = {
     await execa('yarn', ['install'])
   },
 }
+
+export enum AuthSetupMode {
+  FORCE = 'force', // user passed the --force flag, this is essentially replace without prompts
+  REPLACE = 'replace', // replace existing auth provider, with the one being setup
+  COMBINE = 'combine', // add the new auth provider along side the existing one(s)
+  UNKNOWN = 'unknown', // we will prompt the user to select a mode
+}
```

Comment:
I'm not sure this is relevant anymore, since this _is_ a TS project, where as CLI isn't. I think the advice on the blog is good for enums/code we expose to the user, but in this case it's only ever used inside the auth helpers, so personally I don't think it's a bad idea. Enums can be a pain when you export a function that only takes an enum for example (which might not work so well with JS maybe)

Happy to change if you feel strongly!

Reply:
I switched to a simple `type`. I didn't see any downsides to it.